submissions/f3d0t/friends-app/app.js

```diff
@@ -0,0 +1,202 @@
+const BODY = document.querySelector("body");
```

Comment:
Why not address to it with `document.body`? 🤔

Reply:
oops)